torch/csrc/autograd/init.cpp

```diff
@@ -159,7 +159,11 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
         return e.deviceType();
       })
       // correlation id of a linked event
-      .def("linked_correlation_id", &KinetoEvent::linkedCorrelationId);
+      .def("linked_correlation_id", &KinetoEvent::linkedCorrelationId)
+      // compute flops
+      .def("flops", [](const KinetoEvent& e) {
+        return e.flops();
```

Comment:
I think you can just use `&KinetoEvent::flops` like above

Reply:
I have tried `&KinetoEvent::flops`, but clang-tidy will throw error saying there is no matching `flops` function.